.github/workflows/pythonpackage.yml

```diff
@@ -58,6 +58,7 @@ jobs:
               python-build
               flake8
               pytest
+              numpy<2
```

Comment:
Do we still want this pinned?

Reply:
Unfortunately, the combination of `numpy<2, gdal>=3.8` appears necessary with the latest changes to conda-forge builds.  I have been trying to figure out the right way around this, but it is elusive.  So, as a workaround, I have added `numpy<2` to the requirements here.

Reply:
Ok! I would just suggest adding a comment explaining that so we know if we can remove it in the future